css/style.css

```diff
@@ -1,5 +1,127 @@
+* {
```

Comment:
These styles should probably be applied to `body` instead of `*`. The `*` selector will go though and apply these styles to each and every element individually. On the other hand, the `body` selector will only apply the styles to the `body`, but each element will inherit from it , which is significantly faster.